test/platform.test.ts

```diff
@@ -5,6 +5,7 @@
 import * as assert from "assert";
 import * as vscode from "vscode";
 import * as platform from "../src/platform";
+import { SessionManager } from "../src/session";
```

Comment:
Do you have to import this in order to pass undefined into the function call?

Reply:
No I don't think so.  That was probably a left-over line change from a previous approach.  I can remove it.